08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,101 @@
-/*Add CSS rules*/
\ No newline at end of file
+* {
+  -webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+    -moz-box-sizing: border-box;    /* Firefox, other Gecko */
+    box-sizing: border-box;         /* Opera/IE 8+ */
+
+    font-family: 'Open Sans', sans-serif;
+}
+
+.clearfix:after {
+	visibility: hidden;
+	display: block;
+	font-size: 0;
+	content: " ";
+	clear: both;
+	height: 0;
+	}
+* html .clearfix             { zoom: 1; } /* IE6 */
+*:first-child+html .clearfix { zoom: 1; } /* IE7 */
+
+body.nowrap {
+  width: 1200px;
+  overflow: scroll;
+
+}
+
+.mainpanel  {
+width: 400px;
+height: 470px;
+border: 1px dotted gray;
+float: left;
+}
+
+mainpanel:nth-last-child(2) {
+  clear: left;
+}
+
+.truckimg {
+width: 360px;
+height: 200px;
+margin: 15px 15px 40px;
+background-image: url("http://127.0.0.1:3000/08LessonEight/homework/img/truck.jpg");
```

Comment:
the ip number is unique to your computer; this file path isn't hosted on a public server, so no one else will be able to see this!